src/cryptoadvance/specter/managers/config_manager.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, data_folder, config={}):
             "hwi_bridge_url": "/hwi/api/",
             # unique id that will be used in wallets path in Bitcoin Core
             # empty by default for backward-compatibility
```

Comment:
Not empty anymore :)

Reply:
I don't get that comment. We don't need any backward-compatibility if a `config.json` has not been created yet. But good spot, i'll remove that comment somewhere else.
Thanks.